lib/yellow-sdk/client.rb

```diff
@@ -11,49 +11,56 @@ def initialize(api_key, api_secret)
             @api_secret = api_secret
         end
 
-        def create_invoice(base_ccy, base_price, callback)
+        def create_invoice(hash={})
```

Comment:
this is the standard approach of dynamic arguments in ruby. Starting from Ruby v2.0.0 we can use `**kwargs` just like in python, but it seems that approach is still not that common yet. And many ruby devs still use Ruby 1.9. So I think this is good to go.